google/resource_google_project_services.go

```diff
@@ -71,6 +74,7 @@ func resourceGoogleProjectServicesRead(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	d.Set("project", d.Id())
```

Comment:
Should `project` content be checked before overwriting?

Reply:
Eh I don't think it needs to be but may as well throw an error